main.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from urllib.parse import urlparse 
+
+Apache_Type = ["text/plain", "text/plain; charset=ISO-8859-1", 
```

Comment:
I would name this `ApacheType` or `apache_type`. Additionally, should it be "private" (underscored)?

Reply:
Actually, being a non-mutable constant, maybe `_APACHE_TYPES` makes more sense?